lib/mayaUsd/fileio/utils/meshWriteUtils.h

```diff
@@ -73,6 +73,15 @@ namespace UsdMayaMeshUtil
     MAYAUSD_CORE_PUBLIC
     void assignSubDivTagsToUSDPrim(MFnMesh&, UsdGeomMesh&, UsdUtilsSparseValueWriter&);
 
+    MAYAUSD_CORE_PUBLIC
+    void writeVertexData(const MFnMesh&, UsdGeomMesh&, const UsdTimeCode&, UsdUtilsSparseValueWriter&);
```

Comment:
Can we all this `writePointsData()` for consistency with the USD naming that we're actually authoring?

Reply:
@mattyjams see a02159ccad0f0b0165f0214f1710f7680e67561e